geonode/upload/forms.py

```diff
@@ -154,6 +160,14 @@ def _get_files_paths_or_objects(self, cleaned_data):
 
         return uploaded, files
 
+    def validate_parallelism_limit_per_user(self):
+        max_parallel_uploads = self._get_max_parallel_uploads()
+        parallel_uploads_count = self._get_parallel_uploads_count()
+        if parallel_uploads_count >= max_parallel_uploads:
```

Comment:
It could be a problem to have the current parallel uploads at the same as max uploads?
So having this: `if parallel_uploads_count > max_parallel_uploads` could raise many issues?

Reply:
in this case, this means that: (in the case of a limit of 5)
* We already have 5 active uploads.
* This form validation is called before creating one more upload.
* If we already have the limit and we create another one, we will end with 6 active uploads.

So I think it will be better to keep the `>=` instead of `>`.

Reply:
great, thanks for the clarification